src/migrations/__tests__/simple_migration.js

```diff
@@ -0,0 +1,212 @@
+        try {
+          await invalidSimpleMigration._upgradeObject(dummySavedObject);
+        } catch (e) {
+          return;
+        }
+        expect().fail('Exception not thrown!');
+      });
+    });
+    describe('If count: 0', () => {
+      let search;
+      const simpleMigration = new SimpleMigration(configuration);
+      const _shouldObjectUpgradeSpy = sinon.spy(simpleMigration, '_shouldObjectUpgrade');
+      const _upgradeObjectSpy = sinon.spy(simpleMigration, '_upgradeObject');
+      simpleMigration._savedObjectType = savedObjectType;
+      beforeEach(() => {
+        search = sinon.stub(client, 'search').callsFake(function (searchOptions) {
+          return {
+            _scroll_id: 'scroll_id',
+            hits: {
+              total: 0,
+              hits: []
+            }
+          };
+        });
+      });
+      afterEach(() => {
+        search.restore();
+        _shouldObjectUpgradeSpy.resetHistory();
+        _upgradeObjectSpy.resetHistory();
+      });
+      it('_shouldObjectUpgrade not called',async () => {
+        const count = await simpleMigration.count();
+        expect(count).to.be(0);
+        sinon.assert.notCalled(_shouldObjectUpgradeSpy);
+      });
+      it('_upgradeObject not called',async () => {
+        const count = await simpleMigration.upgrade();
+        expect(count).to.be(0);
+        sinon.assert.notCalled(_upgradeObjectSpy);
+      });
+    });
+    describe('If object version > newVersion', () => {
+      let search;
+      const simpleMigration = new SimpleMigration(configuration);
+      simpleMigration._newVersion = 1;
+      const _shouldObjectUpgradeSpy = sinon.spy(simpleMigration, '_shouldObjectUpgrade');
+      const _upgradeObjectSpy = sinon.spy(simpleMigration, '_upgradeObject');
+      simpleMigration._savedObjectType = savedObjectType;
+      beforeEach(() => {
+        search = sinon.stub(client, 'search').callsFake(function (searchOptions) {
+          return {
+            _scroll_id: 'scroll_id',
+            hits: {
+              total: 1,
+              hits: [dummySavedObject]
+            }
+          };
+        });
+      });
+      afterEach(() => {
+        search.restore();
+        _shouldObjectUpgradeSpy.resetHistory();
+        _upgradeObjectSpy.resetHistory();
+      });
+      it('_shouldObjectUpgrade not called',async () => {
+        const count = await simpleMigration.count();
+        expect(count).to.be(0);
+        sinon.assert.notCalled(_shouldObjectUpgradeSpy);
+      });
+      it('_upgradeObject not called',async () => {
+        const count = await simpleMigration.upgrade();
+        expect(count).to.be(0);
+        sinon.assert.notCalled(_upgradeObjectSpy);
+      });
+    });
+    describe('_getSearchParams',() => {
+      let simpleMigration;
+      beforeEach(() => {
+        simpleMigration = new SimpleMigration(configuration);
+      });
+      it('throws exception if both _savedObjectType and _savedObjectType defined', () => {
```

Comment:
_savedObjectType and _savedObjectType ?

Reply:
This type was fixed yesterday https://github.com/sirensolutions/kibiutils/pull/35/commits/e10e12f9099310e06ab6bc79b0d65c7488d2ffe1#diff-9d561434d6abc38780000d37e9ade1ceR353